device/mellanox/x86_64-mlnx_msn2700-r0/plugins/eeprom.py

```diff
@@ -30,3 +31,11 @@ class board(eeprom_tlvinfo.TlvInfoDecoder):
     def __init__(self, name, path, cpld_root, ro):
         self.eeprom_path = "/bsp/eeprom/vpd_info"
         super(board, self).__init__(self.eeprom_path, 0, '', True)
+
+    def decode_eeprom(self, e):
```

Comment:
When issuing show platform syseeprom, can you explain how decode_eeprom is getting called and from where?

Reply:
It is overriding the implementation of `decode_eeprom` in sonic_eeprom/eeprom_base.py.

Reply:
@nikos-github: I haven't tested this change myself, but you shouldn't need to do anything else.

Reply:
@jleveque Doesn't seem to work for me.

Reply:
@mykolaf: Can you provide assistance here?

Reply:
@nikos-github If you want to try it on a running system, you should paste this file into 
/usr/share/sonic/device/<platform>/plugins/eeprom.py, 
e. g. /usr/share/sonic/device/x86_64-mlnx_msn2100-r0/plugins/eeprom.py

No need to delete .pyc under onic_syseeprom, just under /usr/share/sonic/device/<platform>/plugins/

Make sure to change the correct platform's file